src/query_backend.h

```diff
@@ -165,8 +165,28 @@ enum DbTypes {
   VL_MAP_PAIR_INT_STRING_DOUBLE,
   MAP_PAIR_INT_VL_STRING_DOUBLE,
   VL_MAP_PAIR_INT_VL_STRING_DOUBLE,
+
   // append new types only:
+
+  // map<string, vector<double> >
+  MAP_STRING_VECTOR_DOUBLE,
+  MAP_STRING_VL_VECTOR_DOUBLE,
+  VL_MAP_STRING_VECTOR_DOUBLE,
+  MAP_VL_STRING_VECTOR_DOUBLE,
+  MAP_VL_STRING_VL_VECTOR_DOUBLE,
+  VL_MAP_STRING_VL_VECTOR_DOUBLE,
+  VL_MAP_VL_STRING_VECTOR_DOUBLE,
```

Comment:
But this is entirely inter-release.  I think this is okay.  We never promise to be api compatible on develop branch - we promise to on release to release.